README.md

```diff
@@ -38,6 +38,9 @@ Piggieback is compatible with Clojure 1.8.0+, and _requires_ ClojureScript
 **Note** Piggieback 0.3.7 is the first version compatible with nREPL 0.4+. You need
 Piggieback 0.4+ for nREPL 0.6+.
 
+**Note 2** After you've started the nREPL in one of the following ways, you
```

Comment:
I don't really think this is related to the installation part.

Reply:
I could also repeat the advice from the Leiningen section (‘If you're using Leiningen directly, … [s]kip to starting a ClojureScript REPL.’) in the Clojure CLI section. My problem was that the Clojure CLI section says: ‘Then you can simply run a ClojureScript capable nREPL like this: […] Afterwards simply connect to the running server with your favourite nREPL client (e.g. CIDER).’ I had no idea that I now needed to do what is written in Usage. Cursive doesn't even show a REPL prompt and therefore doesn't indicate whether you're working with Clojure or ClojureScript.

All this might be a no-brainer to someone familiar with Piggieback, nREPL and middleware. But I just wanted a ClojureScript REPL in Cursive and didn't want to have to learn about tooling details.

Reply:
Yeah, I understand your point. My remark was mostly about the placement of this. I agree the documentation should be improved.

Reply:
Okay. I'm not sure what I should do now. I could drop Note 2 and instead add a comment at the end of the Clojure CLI section. Does this sound better? I don't think I could reasonably overhaul the whole documentation.

Reply:
Yeah, that sounds good. I'll extend the docs a bit myself afterwards.

Reply:
Force-pushed a new proposal.

Reply:
Thanks for merging!

Reply:
Aw snap! I committed a type. ‘with with’ in the installation section. – I guess you can fix it when you work on the docs.